modules/thermal_hydraulics/include/base/Simulation.h

```diff
@@ -335,16 +335,16 @@ class Simulation : public libMesh::ParallelObject, public LoggingInterface, publ
     std::set<SubdomainName> _subdomain;
     Real _scaling_factor;
   };
-  THMMesh & _mesh;
+  THMMesh & _thm_mesh;
 
   /// Pointer to FEProblem representing this simulation
   FEProblemBase & _fe_problem;
 
   /// The application this is associated with
-  ThermalHydraulicsApp & _app;
+  ThermalHydraulicsApp & _thm_app;
 
   /// The Factory associated with the MooseApp
-  Factory & _factory;
+  Factory & _m_factory;
```

Comment:
I don't know what the "m" means here - just rename to `_thm_factory`.

Reply:
@joshuahansel thanks, the PR has been updated.

Reply:
@joshuahansel The PR should be ready if you don't have additional comments/suggestions.

Reply:
Yes, I'm just invalidating the failing python target and then I'll merge if that passes.